tests/menubar-editor/data/create-test-files.py

```diff
@@ -0,0 +1,124 @@
+import sys
+import os
+import string
+import glob
+
+import optparse
+import subprocess
+import shlex
+import time
+import getopt
+
+def clean(s):
+  return s.replace('""', "'").replace('"', '').replace('  ', ' ').strip()
+
+def getAssertion(a):
+  if len(a) == 0:
+    return ''
+
+  priority = '1'
+
+  if a.find('1:') == 0:
+    a = a[2:]
+
+  if a.find('2:') == 0:
+    priority = '2'
+    a = a[2:]
+
+  if a.find('3:') == 0:
+    priority = '3'
+    a = a[2:]
+
+  a = '      [' + priority + ', "' + a + '"],\n'
+
+  return a
+
+def getSetupTestPageScript(fname):
+
+  code = ''
+
+  if len(fname.strip()):
+    f = open(os.path.join('js', (fname + '.js')), 'r')
+
+    code = 'setupTestPage: function setupTestPage(testPageDocument) {\n'

```

Comment:
this needs to be changed from `testPageDocument` to be `document` in order to be consistent with the scripts.